src/systemctl/systemctl.c

```diff
@@ -2681,7 +2681,7 @@ static int check_triggering_units(sd_bus *bus, const char *name) {
                         continue;
 
                 if (print_warning_label) {
-                        log_warning("Warning: Stopping %s, but it can still be activated by:", n);
+                        log_warning("Warning: %s %s, but it can still be activated by:", disable ? "Disabling" : "Stopping", n);
```

Comment:
I wasn't sure about showing it as "Warning", for the Disabling case. I thought it might be too strong, because it will not always be as much of a problem. I suggested "Notice: " instead.

AIUI there are services where you can choose to either rely on socket activation, or always start them at boot, depending on which units you enable. E.g. both `sshd.socket` and `sshd.service` have [Install] sections.  So you might switch from starting at boot-time, to starting with socket activation, by enabling `sshd.socket` and disabling `sshd.service`.

I think it is relatively harmless to have a message when you do that, but if it says Warning then it might be annoying.

Reply:
I'd also just like to point out that this function checks if the triggering unit is *active*.

This will work fine for the rpcbind case I mentioned in the bug report. But I feel like it's the wrong check. E.g. if you *had* disabled `rpcbind.socket` first, then secondly disabled `rpcbind.service`, then rebooted? You would still get this message on the second step. Even though the reboot achieves your goal of not running rpcbind.

Reply:
OK. I will switch the message from 'Warning' to 'Notice'. That sounds nice. But if it changed, then, what should I (or we?) do for e.g. rpcbind.service?

Reply:
Sorry! I mean, if you run `disable rpcbind.service`, it feels like the right check would be whether `rpcbind.socket` is _enabled_, not whether it is _active_.